Applications/Utils/FileConverter/MeshToRaster.cpp

```diff
@@ -0,0 +1,205 @@
+/**
+ * \file moveMeshNodes.cpp
+ * 2012/03/07 KR Initial implementation
+ *
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/LICENSE.txt
+ */
+
+#include <memory>
+#include <string>
+#include <iostream>
+
+#include <tclap/CmdLine.h>
+
+#include <Applications/ApplicationsLib/LogogSetup.h>
+
+#include "BaseLib/BuildInfo.h"
+#include "GeoLib/AABB.h"
+#include "GeoLib/AnalyticalGeometry.h"
+#include "MeshLib/IO/readMeshFromFile.h"
+#include "MeshLib/Mesh.h"
+#include "MeshLib/Node.h"
+#include "MeshLib/Elements/Tri.h"
+#include "MeshLib/Elements/Quad.h"
+#include "MeshLib/MeshSearch/MeshElementGrid.h"
+
+/// Returns the index of the element p is located in when projected onto a mesh.
```

Comment:
Probably just me, but what is the "element p"?

Reply:
:heavy_check_mark: